packages/react-storage/src/components/StorageBrowser/context/controls/ActionSelect/ActionSelect.tsx

```diff
@@ -22,6 +22,7 @@ export type ActionSelectAction<T = ActionType> =
       type: 'SELECT_ACTION_TYPE';
     }
   | { type: 'EXIT' }
+  | { type: 'ADD_ITEMS'; items: LocationItem[] }
```

Comment:
Does `ADD_ITEMS` make sense and sound okay? If anyone has any suggestions for naming, please let me know